src/BoundaryConditions/fill_halo_regions.jl

```diff
@@ -48,6 +48,10 @@ const MaybeTupledData = Union{OffsetArray, NTuple{<:Any, OffsetArray}}
 function fill_halo_regions!(c::MaybeTupledData, boundary_conditions, indices, loc, grid, args...; fill_open_boundaries = true, kwargs...)
     arch = architecture(grid)
 
+    if fill_open_boundaries
```

Comment:
```suggestion
    if fill_boundary_normal_velocities
```

is this more correct? Do we fill tracer open boundary halos?